cmd/kops/create_cluster.go

```diff
@@ -533,6 +536,10 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		cluster.Spec.CloudProvider = c.Cloud
 
 		if c.Cloud == "vsphere" {
+			if !featureflag.VSphereCloudProvider.Enabled() {
+				return fmt.Errorf("Cloud vSphere is not supported.")
```

Comment:
I am a little confused here. If user set cloud to vsphere but didn't set vsphere feature flag, should we return "Cloud vSphere is not supported"? Or it should be something like "To use vSphere as the cloud provider, please set the KOPS_FEATURE_FLAGS with vSphereCloudProvider"?